### PR TITLE
Adding document properties schemas

### PR DESCRIPTION
**Pull request by alonp99**

### CODE REVIEW

**Review by MatanYadaev — APPROVED** [review]

File: apps/backoffice-v2/src/components/pages/Individual/hooks/useIndividual/useIndividual.tsx [hunk 1]
```diff
@@ -9,6 +8,12 @@ import { useGetSessionQuery } from '../../../../../lib/react-query/queries/useGe
 import { useFilterEntity } from 'hooks/useFilterEntity/useFilterEntity';
 import { components } from 'components/pages/Individual/hooks/useIndividual/components';
 
+const prettifyString = (input: string): string => {
```
[the remark this answers is not in the document]
Comment (alonp99) [reply apps/backoffice-v2/src/components/pages/Individual/hooks/useIndividual/useIndividual.tsx:11]: ✅

File: services/workflows-service/src/documents/schemas/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { getDocumentId } from '@/documents/utils';
+import { ghanaDocuments } from './GH';
+import { Document } from '../types';
+import { countryCodes } from '@/common/countries';
+
+const convertToDictonary = (documents: Document[]) =>
```
[the remark this answers is not in the document]
Comment (alonp99) [reply services/workflows-service/src/documents/schemas/index.ts:6]: ✅

File: services/workflows-service/src/documents/schemas/index.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      const id = getDocumentId(document);
+      // @ts-ignore
+      curr[id] = document;
+
+      return curr;
+    },
+    {} as Record<string, Document>,
+  );
+
+const documentKeysByCountry: Partial<Record<(typeof countryCodes)[number], any>> = {
```
[the remark this answers is not in the document]
Comment (alonp99) [reply services/workflows-service/src/documents/schemas/index.ts:19]: ✅

File: apps/backoffice-v2/src/components/pages/Individual/hooks/useIndividual/useIndividual.tsx [hunk 4]
```diff
@@ -9,6 +8,12 @@ import { useGetSessionQuery } from '../../../../../lib/react-query/queries/useGe
...
+  let words = input.split('_');
+  words = words.map(word => word.charAt(0).toUpperCase() + word.slice(1));
+  return words.join(' ');
```
[the remark this answers is not in the document]
Comment (alonp99) [reply apps/backoffice-v2/src/components/pages/Individual/hooks/useIndividual/useIndividual.tsx:14]: ✅

File: services/workflows-service/src/documents/schemas/index.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  documents.reduce<Record<string, Document>>(
+    // @ts-ignore
+    (curr, document: Document) => {
+      const id = getDocumentId(document);
+      // @ts-ignore
+      curr[id] = document;
+
+      return curr;
+    },
+    {} as Record<string, Document>,
```
[the remark this answers is not in the document]
Comment (alonp99) [reply services/workflows-service/src/documents/schemas/index.ts:16]: Good insight, when thinking about it, it think you're right

**Pull request merged**